en/developer_guide/block_parameter.rst

```diff
@@ -0,0 +1,126 @@
+Block parameter
+===============
+
+Context
+-------
+
+To be properly displayed on the website, some blocks will require environment parameters.
```

Comment:
This sounds a bit heavy and more complicated than it is. Personnally i'd go with "for their display to be customizable, some blocks need to receive parameters when they render"